versions.props

```diff
@@ -3,9 +3,4 @@ junit:junit = 4.12
 one.util:streamex = 0.6.8
 org.assertj:assertj-core = 3.12.1
 org.mockito:mockito-core = 2.24.5
-
-# Conflicts
-com.google.code.findbugs:jsr305 = 3.0.2
-com.google.errorprone:error_prone_annotations = 2.3.3
-org.checkerframework:checker-qual = 2.5.3
 org.objenesis:objenesis = 3.0.1
```

Comment:
Can this be removed as well? We don't explicitly depend on this.

Reply:
when i removed it, the transitive constraints bumped it donw to 2.6, and i wasn't sure if that's safe